docs/source/api.rst

```diff
@@ -0,0 +1,212 @@
+Unstructured API
+#################
+
+Try our hosted API! It's freely available to use with any of the file types listed above. This is the easiest way to get started, all you need is an API key. You can get your API key `here <https://www.unstructured.io/api-key/>`_ now and start using it today.
```

Comment:
Can you also add a link to the API docs page?

https://api.unstructured.io/general/docs